doc/source/yt4differences.rst

```diff
@@ -47,6 +47,27 @@ are being removed entirely. When the deprecated field names are used for the
 first time in a session, a warning will be logged, so it is advisable to set
 your logging level to ``WARNING`` at a minimum to catch these.
 
+Colormaps
+^^^^^^^^^
+yt previously included ``cmocean`` colormaps.  Due to collisions between some
+of our dependencies, you now have to explicitly import the ``cmocean`` module
+and prefix ``cmocean`` colormaps (like ``balance``) with ``cmo.``.  Example:
```

Comment:
```suggestion
yt used to automatically register colormaps from ``cmocean``, unprefixed.
This unfortunately became unsustainable with the 3.4 release of Matplotlib, in which colormaps with colliding
names raise errors. The fix is to explicitly import the ``cmocean`` module and prefix ``cmocean`` colormaps
(like ``balance``) with ``cmo.``.  Note that this solution works with any versions of yt+Matplotlib. Example:
```

Reply:
I originally wrote that your example worked with any version of yt+matplotlib. After working on #3212, I realised this was unfortunately not the case, so I updated this accordingly.